client/stylesheets/_toCat.scss

```diff
@@ -3671,6 +3671,7 @@ On click the images will be displayed at normal size to complete the effect
 
 .profileUserName h1 {
     font-size: 2.2rem;
+    font-weight: 400 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces